src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -39,6 +39,9 @@ public boolean start() throws Exception {
         System.out.println(templates);
 
         Node node = computer.getNode();
+        if (node == null) {
+            throw new Exception("Computer does not correspond to a live node");
```

Comment:
Yes, this might be very tricky to test manually, perhaps only a corner case unit test can manage this? Yes you can change the message on the other PR.

Ok, maybe it is not practical to check for a null `Computer` object inside the plugin.